packages/vscode-extension/src/webview/components/shared/KyebindingInfo.tsx

```diff
@@ -0,0 +1,59 @@
+import { ReactNode, useEffect, useState } from "react";
+import "./KeybindingInfo.css";
+import { useProject } from "../../providers/ProjectProvider";
+
+interface KeybindingInfoProps {
+  commandName: string;
+}
+
+interface SymbolProps {
+  children: ReactNode;
+}
+
+const Symbol = ({ children }: SymbolProps) => {
+  return <span className="symbol">{children}</span>;
+};
+
+export const KeybindingInfo = ({ commandName }: KeybindingInfoProps) => {
+  const { project } = useProject();
+  const [keybinding, setKeybinding] = useState<string[]>([]);
+
+  function translateToUnicode(symbol: string) {
```

Comment:
this method doesn't need to be defined inside of the component or am I missing something?